### ISSUE TITLE
global routing failing with routing congestion high

### ISSUE THREAD

**Issue by vijayank88** [The-OpenROAD-Project/OpenROAD#1620]
Using OpenLane flow latest tag for this run.

Used relative die size calculated by yosys. Used full core utilization still flow failing with routing congestion too high error.

Attached reproducible tar file for reference.
[openroad_issue_reproducible.zip](https://github.com/The-OpenROAD-Project/OpenROAD/files/8005086/openroad_issue_reproducible.zip)
MPW repo: https://github.com/yongatek/caravel_yonga-turbo-encoder.git

logs:
```

[INFO GRT-0053] Routing resources analysis:
          Routing      Original      Derated      Resource
Layer     Direction    Resources     Resources    Reduction (%)
---------------------------------------------------------------
li1        Vertical            0             0          0.00%
met1       Horizontal    4250400       1722627          59.47%
met2       Vertical      3187800       2128898          33.22%
met3       Horizontal    2125200       1487160          30.02%
met4       Vertical      1275120        834490          34.56%
met5       Horizontal     425040        212058          50.11%
---------------------------------------------------------------

[INFO GRT-0101] Running extra iterations to remove overflow.
[INFO GRT-0103] Extra Run for hard benchmark.
[INFO GRT-0197] Via related to pin nodes: 305834
[INFO GRT-0198] Via related Steiner nodes: 10805
[INFO GRT-0199] Via filling finished.
[INFO GRT-0111] Final number of vias: 489389
[INFO GRT-0112] Final usage 3D: 2797771
[ERROR GRT-0118] Routing congestion too high.
Error: groute.tcl, 69 GRT-0118
```

**Comment by eder-matheus** [comment]
@vijayank88 I was able to run the global route in this design by reducing the global adjustment (`GLB_RT_ADJUSTMENT`) from `0.3` to `0.25`.  Do you know if the global placement is using the `routability_driven` flag? Perhaps we just need a slightly better placement to make the global route finishes properly with the current configuration.

**Comment by vijayank88** [comment]
@eder-matheus 
I've tested with OR commit `OpenROAD 3eafa6da2942ec22d27e2208c99c7a449af34fbf` and faced congestion issue with OpenLane tag: openlane 2022.02.10_02.15.04
in OpenLane `set ::env(PL_ROUTABILITY_DRIVEN) {0}` default set. 
With latest tags OR commits got updated. Let me try again with changes and update you on this.
Thanks...

**Comment by vijayank88** [comment]
@eder-matheus 
By reducing the global adjustment (GLB_RT_ADJUSTMENT) from 0.3 to 0.25 resolved the congestion issue.
